backend/maelstro/main.py

```diff
@@ -121,6 +135,14 @@ def get_destinations() -> list[dict[str, str]]:
     return config.get_destinations()
 
 
+@app.get("/transformations")
+def get_transformations(with_src_dst: bool = False) -> dict[str, Any]:
+    return {
```

Comment:
not sure this is really needed.

Reply:
At some time it could be interesting showing this in frontend, but probably only for a pair of source and destination.

Reply:
we'll harmonize this with the clone_preview entry point